rdrf/registry/patients/migrations/0037_patient_stage_data_migration.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 2.1.9 on 2019-06-18 14:53
+
+from django.db import migrations, models
+
+
+def add_patient_stages(apps, schema_editor):
```

Comment:
We don't set up data in data migrations in this project, we use "initial data" instead.
See: https://github.com/eresearchqut/trrf/tree/next_release/rdrf/rdrf/initial_data

Reply:
Updated.